installer/install.iss

```diff
@@ -2245,45 +2250,47 @@ begin
 
     // Restore the settings chosen during a previous install
     RdbExperimentalOptions[GP_BuiltinDifftool].Checked:=ReplayChoice('Enable Builtin Difftool','Auto')='Enabled';
 #endif
 
 #ifdef WITH_EXPERIMENTAL_BUILTIN_REBASE
-    // 2nd option
     RdbExperimentalOptions[GP_BuiltinRebase]:=CreateCheckBox(ExperimentalOptionsPage,'Enable experimental, built-in rebase','<RED>(NEW!)</RED> Use the experimental built-in rebase (about 70% faster, but only'+#13+'lightly tested).',TabOrder,Top,Left);
 
     // Restore the settings chosen during a previous install
     RdbExperimentalOptions[GP_BuiltinRebase].Checked:=ReplayChoice('Enable Builtin Rebase','Auto')='Enabled';
 #endif
 
 #ifdef WITH_EXPERIMENTAL_BUILTIN_STASH
-    // 3rd option
     RdbExperimentalOptions[GP_BuiltinStash]:=CreateCheckBox(ExperimentalOptionsPage,'Enable experimental, built-in stash','<RED>(NEW!)</RED> Use the experimental built-in stash (about 90% faster, but only'+#13+'lightly tested).',TabOrder,Top,Left);
 
     // Restore the settings chosen during a previous install
     RdbExperimentalOptions[GP_BuiltinStash].Checked:=ReplayChoice('Enable Builtin Stash','Auto')='Enabled';
 #endif
 
 #ifdef WITH_EXPERIMENTAL_BUILTIN_ADD_I
-    // 4th option
     RdbExperimentalOptions[GP_BuiltinAddI]:=CreateCheckBox(ExperimentalOptionsPage,'Enable experimental, built-in add -i/-p','<RED>(NEW!)</RED> Use the experimental built-in interactive add ("git add -i" or "git add -p").'+#13+'This makes it faster (especially the startup!), but it is not yet considered robust.',TabOrder,Top,Left);
 
     // Restore the settings chosen during a previous install
     RdbExperimentalOptions[GP_BuiltinAddI].Checked:=ReplayChoice('Enable Builtin Interactive Add','Auto')='Enabled';
 #endif
 
 #ifdef WITH_EXPERIMENTAL_PCON
-    // 5th option
     RdbExperimentalOptions[GP_EnablePCon]:=CreateCheckBox(ExperimentalOptionsPage,'Enable experimental support for pseudo consoles.','<RED>(NEW!)</RED> This allows running native console programs like Node or Python in a'+#13+'Git Bash window without using winpty, but it still has known bugs.',TabOrder,Top,Left);
 
     // Restore the settings chosen during a previous install
     RdbExperimentalOptions[GP_EnablePCon].Checked:=ReplayChoice('Enable Pseudo Console Support','Auto')='Enabled';
 #endif
 
+#ifdef WITH_EXPERIMENTAL_SCALAR
+    RdbExperimentalOptions[GP_InstallScalar]:=CreateCheckBox(ExperimentalOptionsPage,'Install Scalar (ported to C)','<RED>(NEW!)</RED> Install Scalar (ported from <A HREF=https://github.com/microsoft/scalar#readme>the .NET-based version</A>).',TabOrder,Top,Left);
+
+    // Restore the settings chosen during a previous install
+    RdbExperimentalOptions[GP_InstallScalar].Checked:=ReplayChoice('Install Scalar','Auto')='Disabled';
```

Comment:
We want this to be enabled by default, at least for `microsoft/git`, right?

Reply:
Yes, eventually. The way to do this is to pass `/o:InstallScalar=Enabled` to the installer on the command-line. Or to invert the logic where `Auto` would imply `Disabled`.

Reply:
Or maybe you want me to flip the default to enable it already, in preparation for an experimental release including the native Scalar?

Reply:
Yeah, I'm actually thinking we shouldn't even have the option: it's either included or not. At least, that should be the case for `microsoft/git` and I'm not sure the best course of action for `git-for-windows/git`.

Reply:
I could also "hide" it under Components, next to Git LFS, defaulting to "on". What do you think?

Reply:
That level of fine-grained detail might be enough for us to be confident it will be installed without alienating power users. Thanks.

Reply:
Done.